pkg/plan/recipe/install_plans.go

```diff
@@ -108,6 +111,17 @@ func BuildCRIPlan(criSpec *baremetalspecv1.ContainerRuntime, cfg *envcfg.EnvSpec
 			plan.DependOn("install:docker"))
 	}
 
+	if IsDockerOnCentOS {
```

Comment:
This duplicates what is in BuildK8SPlan.  Should that depend on install:docker or should that other block be deleted?

Reply:
Thoughts?

Reply:
This isn't exactly a duplicate. The original case _conditionally_ adds the resource based on a probe of the machine environment:

Reply:
I'm addressing this change. thank you @jrryjcksn !

Reply:
I'm fixing the logic. But these 2 selimux:permissive steps are *slightly* different.